automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebookPythonKernelSpec.scala

```diff
@@ -0,0 +1,151 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+import java.io.File
+
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.dao.Google.googleStorageDAO
+import org.broadinstitute.dsde.workbench.fixture.BillingFixtures
+import org.broadinstitute.dsde.workbench.model.google.{EmailGcsEntity, GcsEntityTypes, GcsObjectName, GcsRoles, GoogleProject}
+import org.broadinstitute.dsde.workbench.service.test.RandomUtil
+import org.broadinstitute.dsde.workbench.service.{Orchestration, Sam}
+import org.scalatest.{BeforeAndAfterAll, FreeSpec}
+
+import scala.language.postfixOps
+
+class NotebookPythonKernelSpec extends FreeSpec with LeonardoTestUtils with BeforeAndAfterAll with BillingFixtures with RandomUtil {
```

Comment:
Maybe rename to `NotebookKernelSpec` since this exercises both python and R kernels?

Alternatively should we have a separate class for `NotebookRKernelSpec`?